tests/monster_test_generated.rs

```diff
@@ -9,6 +9,7 @@ use std::mem;
 use std::cmp::Ordering;
 
 extern crate flatbuffers;
+extern crate bitflags;
```

Comment:
This is not good.
Either generate bitflags without any dependency, or put this into `flatbuffers` lib and reexport with
```rust
pub use bitflags
```
so this generated code would look like
```rust
extern crate flatbuffers;
use flatbuffers::{bitflags, /* other */ };
```

Reply:
Done